doc/development-context-aware-applications/deep-dive.md

```diff
@@ -1,12 +1,5 @@
 Should you wish to learn much more about this, please check: 
 
    - [Introduction to NGSI API and Orion Context Broker](http://bit.ly/fiware-orion)
-   - [Github repository (reference implementation) ](https://github.com/Fiware/context.Orion)
-   - [Play with the Docker container](https://hub.docker.com/r/fiware/orion/)
-   - [Browse NGSI v1](http://fiware.github.io/context.Orion/api/v1/)
-   - [Browse NGSI v2](http://fiware.github.io/specifications/ngsiv2/latest/)
```

Comment:
why are you removing all this content?

Reply:
again, please do not remove this, the deep dive section is intended to provide all the links which provide extra resources, so again a big -1 to this change.